src/main/kotlin/no/nav/familie/ef/mottak/task/OpprettSakTask.kt

```diff
@@ -9,30 +11,43 @@ import no.nav.familie.prosessering.domene.TaskRepository
 import org.slf4j.LoggerFactory
 import org.springframework.data.repository.findByIdOrNull
 import org.springframework.stereotype.Service
+import java.time.LocalDateTime
+import java.time.LocalTime
 
+/**
+ * Infotrygd er vanligvis stengt mellom 21 og 6, men ikke alltid.
+ * Hvis tasken feiler i denne tida så lager den en ny task og kjører den kl 06
+ */
 @Service
 @TaskStepBeskrivelse(taskStepType = OpprettSakTask.TYPE,
                      beskrivelse = "Oppretter sak i Infotrygd")
 class OpprettSakTask(private val taskRepository: TaskRepository,
                      private val sakService: SakService,
+                     private val dateTimeService: DateTimeService,
                      private val soknadRepository: SoknadRepository) : AsyncTaskStep {
 
     private val logger = LoggerFactory.getLogger(this::class.java)
 
     override fun doTask(task: Task) {
-        val oppgaveId: String? = task.metadata[LagBehandleSakOppgaveTask.behandleSakOppgaveIdKey] as String?
-        oppgaveId?.let {
-            val sakId = sakService.opprettSak(task.payload, it)?.trim()
+        val oppgaveId: String = task.metadata[LagBehandleSakOppgaveTask.behandleSakOppgaveIdKey] as String?
```

Comment:
Vi kan vel ikke vite om det finnes en oppgaveId i metadata? 
Denne settes vel bare for noen tasker?

Reply:
Ref: no.nav.familie.ef.mottak.task.LagBehandleSakOppgaveTask#doTask
 if (sakService.kanOppretteInfotrygdSak(soknad)) {
            val lagBehandleSakOppgave = oppgaveService.lagBehandleSakOppgave(journalpost)
            task.metadata.apply {
                this[behandleSakOppgaveIdKey] = lagBehandleSakOppgave.toString()
            }
                taskRepository.save(task)
        }

Reply:
Nja - den bør kanskje være der...